lib/event-bus.js

```diff
@@ -16,11 +16,14 @@ module.exports = class EventBus {
         return;
       }
 
-      const evnt = JSON.parse(newValue);
+      const evnt = newValue && JSON.parse(newValue);
```

Comment:
Yeah stdout is tricky over the Chrome Devtools Protocol, which is why I went with the event bus through local storage. It's not ideal by any means and there are still [issues with it](https://github.com/webpack/webpack/blob/master/lib/HotModuleReplacementPlugin.js).